### PR TITLE
detail page empty with tensorflow tutorial code because of the "None"

### PR DESCRIPTION
**Pull request by Lijiaoa**
### Description ###
**should be merged after the PR #4990 !!!** 

**fix issue:** #4969 
**background:** for hpo experiment: its search space has "None"(in the yaml file) value and trial parameters have `null`(in ts) value
**cause bug:** webui hyper-parameter chart will broke down because the `null`
**resolve solution:** convert the `null` to string 'null' in code

### How to test ###
as the issue to run an hpo experiment, and when having succeeded trials to view hyper-parameter chart. If the chart is normal and the page is not broken down, the fix PR worked fine~

### CODE REVIEW

**Review by ultmaster — APPROVED** [review]

File: ts/webui/src/static/function.ts [hunk 1]
```diff
@@ -254,7 +254,12 @@ function formatComplexTypeValue(value: any): string | number {
     if (['number', 'string'].includes(typeof value)) {
         return value;
     } else {
-        return value.toString();
+        // for hpo experiment: search space choice value is None, and it shows null
+        if (Object.is(null, value)) {
```
Comment (liuzhe-lz) [inline ts/webui/src/static/function.ts:258]: I think `String(value)` is more general and robust.

Comment (Lijiaoa) [reply ts/webui/src/static/function.ts:258]: ok! string() is better than toString

File: ts/webui/src/static/model/trial.ts [hunk 2]
```diff
@@ -234,6 +234,12 @@ class Trial implements TableObj {
         } else {
             const tempHyper = this.info.hyperParameters;
             let params = JSON.parse(tempHyper[tempHyper.length - 1]).parameters;
+            // for hpo experiment: search space choice value is None, and it shows null
+            for (const key in params) {
```
Comment (liuzhe-lz) [inline ts/webui/src/static/model/trial.ts:238]: Use for-of if there is not special reason.

Comment (Lijiaoa) [reply ts/webui/src/static/model/trial.ts:238]: yes!

**Review by liuzhe-lz — APPROVED** [review]

**Pull request merged**